src/main/java/fr/openmc/core/utils/messages/MessagesManager.java

```diff
@@ -47,61 +49,29 @@ public static void sendMessage(CommandSender sender, String message, Prefix pref
         String messageStr = prefix.getPrefix() + " §7» " + message;
 
         sender.sendMessage(messageStr);
-
-    }
-
-
-    private static String getPrefixType(MessageType type) {
-        return switch (type) {
-            case ERROR -> "§c❗";
-            case WARNING -> "§6⚠";
-            case SUCCESS -> "§a✔";
-            case INFO -> "§bⓘ";
-            default -> "§7";
-        };
-    }
-
-    private static Sound getSound(MessageType type) {
-        return switch (type) {
-            case ERROR, WARNING -> Sound.BLOCK_NOTE_BLOCK_BASS;
-            case SUCCESS -> Sound.BLOCK_NOTE_BLOCK_BELL;
-            case INFO -> Sound.BLOCK_NOTE_BLOCK_BIT;
-            default -> null;
-        };
     }
 
     public static String textToSmall(String text) {
         StringBuilder result = new StringBuilder();
+        Map<Character, Character> charMap = new HashMap<>();
+
         String smallLetters = "ᴀʙᴄᴅᴇꜰɢʜɪᴊᴋʟᴍɴᴏᴘǫʀѕᴛᴜᴠᴡхʏᴢ";
         String normalLetters = "abcdefghijklmnopqrstuvwxyz";
-        String normalLettersCaps = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
         String numbers = "₁₂₃₄₅₆₇₈₉₀";
         String numbersNormal = "1234567890";
 
-        if (text.contains("§")) {
-            String[] split = text.split("§");
-            for (int i = 0; i < split.length; i++) {
-                if (i == 0) {
-                    result.append(split[i]);
-                    continue;
-                }
-                if (split[i].length() > 1) {
-                    result.append("§").append(split[i].charAt(0)).append(textToSmall(split[i].substring(1)));
-                } else {
-                    result.append("§").append(split[i]);
-                }
-            }
-            return result.toString();
+        for (int i = 0; i < 26; i++) {
+            charMap.put(normalLetters.charAt(i), smallLetters.charAt(i));
+            charMap.put(normalLetters.charAt(i + 26), smallLetters.charAt(i));
+        }
+        for (int i = 0; i < numbersNormal.length(); i++) {
+            charMap.put(numbersNormal.charAt(i), numbers.charAt(i));
         }
```

Comment:
Peut-être faire un truc dans le genre suivant pour éviter une boucle de plus
```suggestion
        for (int i = 0; i < 26; i++) {
            charMap.put(normalLetters.charAt(i), smallLetters.charAt(i));
            charMap.put(normalLetters.charAt(i + 26), smallLetters.charAt(i));
            
            if (i < 10) {
                charMap.put(numbersNormal.charAt(i), numbers.charAt(i));
            }
        }
```

Reply:
Tu peux utiliser un BiMap
[bi-directional-map-in-java](https://stackoverflow.com/questions/10699492/bi-directional-map-in-java)